php-library/templates/_base.ingress.tpl

```diff
@@ -28,6 +28,12 @@ alb.ingress.kubernetes.io/healthcheck-port: {{ $ingress.healthcheckPort | defaul
 alb.ingress.kubernetes.io/healthcheck-interval-seconds: {{ $ingress.healthcheckInterval | default 30 | quote }}
 alb.ingress.kubernetes.io/success-codes: {{ $ingress.healthcheckSuccessCodes | default "200" | quote }}
 {{- end }}
+{{- if $ingress.waf | default false }}
+alb.ingress.kubernetes.io/wafv2-acl-arn: {{ $ingress.wafAclArn | quote }}
+{{- end }}
+{{- if $ingress.albgroup | default false }}
+alb.ingress.kubernetes.io/group.name: {{ include "phplibrary.base.ingress.annotations.tags" (list $top $ingress) | quote }}
```

Comment:
Wat zou hier de uitkomst moeten zijn? Gewoon 1 string toch lijkt me?

Reply:
Klopt, het is één string, maar komma gescheiden.

Maar ik bedenk me net inderdaad dat zoiets niet handig is in een ALB group name, dus ik ga het anders doen.